cloud/vsphere/provisioner/govmomi/utils.go

```diff
@@ -1,4 +1,4 @@
-package common
```

Comment:
The main reason I had it separate was if in future we have more that 1 actuator implementations (or `variants` implementations) then this would serve as the common code that could be reused across. Essentially terraform actuator could have used this as well but since we were going to remove that anyways, thus I did not make changes in that to use this common code.
Anyways, It's fine for now but later on we may still end up re-creating something like this again based on the different `variant` implementations.